.github/workflows/cpp-lint.yml

```diff
@@ -9,8 +9,15 @@ name: C++ Lint
 
 on:
   pull_request:
-    branches: [master, nightly]
-    types: [opened, synchronize, reopened]
+    branches:
+      - master
+    types:
+      - opened
+      - synchronize
+      - reopened
+  push:
+    branches:
+      - master
```

Comment:
See the top of this file.

Reply:
Ugh, RIP my alphabet skills...

Reply:
Hmmm, can you make it so that the linter is run on push? I often find myself opening PRs with some linter errors.

Reply:
I'd rather not as we already have a total of 92 workflows that need to run on push into master... and we're limited to 20 concurrent... and we know if this has passed on the PR then there is no way it could fail on the push.

![image](https://github.com/LizardByte/libdisplaydevice/assets/42013603/14035d37-4a90-4ee5-b7e8-49420a5bccfb)

You can run this python script though? https://github.com/LizardByte/libdisplaydevice/blob/master/scripts/update_clang_format.py